community.html

```diff
@@ -34,7 +34,7 @@ <h2>Help others</h2>
     <li class="talk"><h3>Give a Talk</h3>
       <p>Get started with our <a href="https://github.com/sandstorm-io/sandstorm/wiki/Speaker-Kit-Lightning-Talk">speaker kit</a> to prepare your presentation. <a href="https://groups.io/g/sandstorm-user-group">Send us an email</a> if you need help with finding a venue or preparing for your talk.</p>
     <li class="discuss"><h3>Discuss &amp; Answer Questions</h3>
-      <p>User help happens on the <a href="https://groups.io/g/sandstorm-user-group">mailing list</a>; join us and contribute! Real-time chat happens on <a href="https://matrix.to/#/#sandstorm:libera.chat">Matrix</a>. You can see old IRC <a href="https://libera.irclog.whitequark.org/sandstorm/">chat archives</a> too.</p>
+      <p>User help happens on the <a href="https://groups.io/g/sandstorm-user-group">mailing list</a>; join us and contribute! Real-time chat happens on <a href="https://sandstorm.zulipchat.com">Zulip chat</a>. You can see old IRC <a href="https://libera.irclog.whitequark.org/sandstorm/">chat archives</a> or Matrix <a href="https://matrix.to/#/#sandstorm:libera.chat"> too.</p>

```

Comment:
Good call but I'd change the wording slightly.

"Zulip chat" is good in other places but here I think I'd just say "Zulip".

And for "or Matrix" I'd probably say "or our old Matrix room". Also the link tag to our Matrix room is misplaced, and doesn't have a closing tag. I'd probably have the link be on "Matrix room".

Reply:
Actually maybe "our Zulip"? Since they're technically separate instances? Your call, or maybe someone else has an opinion.